packages/main/src/Slider.ts

```diff
@@ -276,40 +264,7 @@ class Slider extends SliderBase implements IFormInputElement {
 		this._valueOnInteractionStart = this.value;
 	}
 
-	_onInputFocusOut(e: FocusEvent) {
-		const tooltipInput = this.shadowRoot!.querySelector("[ui5-input]") as Input;
-
-		this._tooltipVisibility = SliderBase.TOOLTIP_VISIBILITY.HIDDEN;
-		this._updateValueFromInput(e);
-
-		if (!this._isInputValueValid) {
-			tooltipInput.value = this._lastValidInputValue;
-			this._isInputValueValid = true;
-			this._tooltipInputValueState = "None";
-		}
-	}
-
-	_updateInputValue() {
-		const tooltipInput = this.shadowRoot!.querySelector("[ui5-input]") as Input;
-
-		if (!tooltipInput) {
-			return;
-		}
-
-		this._isInputValueValid = parseFloat(tooltipInput.value) >= this.min && parseFloat(tooltipInput.value) <= this.max;
-
-		if (!this._isInputValueValid) {
-			this._tooltipInputValue = this._lastValidInputValue;
-			this._isInputValueValid = true;
-			this._tooltipInputValueState = "Negative";
-
-			return;
-		}
-
-		this._tooltipInputValue = this.value.toString();
-		this._lastValidInputValue = this._tooltipInputValue;
-		this._tooltipInputValueState = "None";
-	}
+	_updateInputValue() {}
```

Comment:
thats not used anymore, right? we can remove it

Reply:
Done.